Source/Events.Store.MongoDB/Streams/EventsToStreamsWriter.cs

```diff
@@ -14,37 +14,39 @@ namespace Dolittle.Runtime.Events.Store.MongoDB.Streams
     /// <summary>
     /// Represents an implementation of <see cref="IWriteEventsToStreams" />.
     /// </summary>
-    public class EventsToStreamsWriter : IWriteEventsToStreams
+    public class EventsToStreamsWriter : IEventsToStreamsWriter, IWriteEventsToStreams
```

Comment:
Renaming of the interfaces would probably makes this a bit clearer whats going on in here following the convention of adding `Store/Runtime` to the interface class/interface name depending on which they work on.

The `IWriteEventsToStreams` interface is not part of the PR so can't comment on it but it should also then get renamed.
```suggestion
    public class EventsToStreamsWriter : IWriteStoreEventsToStreams, IWriteRuntimeEventsToStreams
```

Reply:
I don't like the "StoreEvents" part. I would counter suggest this:
```suggestion
    public class EventsToStreamsWriter : IWriteEventsToStreamCollection, IWriteEventsToStreams
```

Reply:
Sounds good :+1: